custom/icds_reports/utils/aggregation_helpers/aww_incentive.py

```diff
@@ -27,22 +27,24 @@ def aggregation_query(self):
           SELECT
             %(state_id)s AS state_id,
             %(month)s AS month,
-            awc_id,
-            block_id,
-            state_name,
-            district_name,
-            block_name,
-            supervisor_name,
-            awc_name,
-            aww_name,
-            contact_phone_number,
-            wer_weighed,
-            wer_eligible,
-            awc_num_open,
-            valid_visits,
-            expected_visits
-          FROM agg_ccs_record_monthly AS acm
-          WHERE acm.month = %(month)s AND acm.state_id = %(state_id)s and aggregation_level=5
+            awcm.awc_id,
+            awcm.block_id,
+            awcm.state_name,
+            awcm.district_name,
+            awcm.block_name,
+            awcm.supervisor_name,
+            awcm.awc_name,
+            awcm.aww_name,
+            awcm.contact_phone_number,
+            awcm.wer_weighed,
+            awcm.wer_eligible,
+            awcm.awc_num_open,
+            ccsm.valid_visits,
+            ccsm.expected_visits
+          FROM agg_ccs_record_monthly AS ccsm
```

Comment:
Is agg_ccs_record_monthly aggregated by things like trimester and ccs_status? In that case I believe that you will have several lines for every AWC, but this table looks like it should be one per AWC. Would the following query be a more accurate representation of what is needed:

```
SELECT agg_awc.awc_id, ..., sum(agg_ccs.valid_visits), sum(agg_ccs.expected_visits)
FROM agg_awc
JOIN agg_ccs
WHERE ....
GROUP BY agg_awc.awc_id
```